packages/x-utils/package.json

```
@@ -0,0 +1,35 @@
+{
+  "name": "@empathyco/x-utils",
+  "version": "0.1.0-alpha.0",
+  "description": "A utility package for Empathy search",
+  "author": "Empathy Systems Corporation S.L.",
+  "license": "Apache-2.0",
+  "homepage": "https://github.com/empathyco/x/tree/main/packages/x-utils#readme",
+  "keywords": [
+    "search",
+    "utils"
+  ],
+  "main": "dist/cjs/index.js",
+  "module": "dist/esm/index.js",
```

Add missing property `types` (this is "synchronise with the `ts.config.json` `declarationDir` property)

```suggestion
  "module": "dist/esm/index.js",
  "types": "types/index.d.ts",
```